adam/visualization/panda3d_interface.py

```diff
@@ -174,33 +200,36 @@ def add_model(
         # nested
         else:
             new_model.reparentTo(parent)
-        new_model.setPos(position[0], position[1], position[2])
+        # new_model.setPos(position[0], position[1], position[2])
```

Comment:
This comment looks like it could be removed unless there is another reason for it to stay here?